### PR TITLE
BitZeny: Bump version to 2.0.1.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -3,7 +3,7 @@ AC_PREREQ([2.60])
 define(_CLIENT_VERSION_MAJOR, 2)
 define(_CLIENT_VERSION_MINOR, 0)
 define(_CLIENT_VERSION_REVISION, 0)
-define(_CLIENT_VERSION_BUILD, 0)
+define(_CLIENT_VERSION_BUILD, 1)
```
Comment (hitobb) [inline configure.ac]: Please count up (_CLIENT_VERSION_REVISION) insted of (_CLIENT_VERSION_BUILD).
As build versions are often omitted,it is not the best

**Comment by ghost** [comment]
@hitobb 
aaa
bbbb
cccc

**Review by hitobb — DISMISSED** [review]
Thanks!
LGTM

**Comment by ghost** [comment]
@BitzenyCoreDevelopers/team rebased. check it again.

**Review by onokatio — APPROVED** [review]
OK

**Pull request merged**